CTPPS Pixel DQM: add 1 histogram (backport of PR 18835)

popovvp opened the pull request:
add new plot rpot occupancy per BX

ikrav committed:
Bug fixes, added debug statements

ikrav committed:
Fixed AOD vs miniAOD detection

ikrav committed:
The first version of 80X cut-based photon ID

ikrav committed:
Added the possibility of having different egmPhotonIDSequence for AOD and miniAOD

Conflicts:
	RecoEgamma/PhotonIdentification/python/egmPhotonIDs_cff.py

ikrav committed:
Adjusted python code that deals with different VID sequences for AOD and miniAOD

ikrav committed:
New photon MVA class for 80X releases

ikrav committed:
Another adjustment for the mechanism of having different sequences in VID for AOD and miniAOD

ikrav committed:
Python configuration for the new photon MVA, imcomplete

ikrav committed:
Removed using of shower shapes from PhotonValueMapProducer that was in use until 753. Now shower shapes are taken directly from the photon object

ikrav committed:
More changes for 80X photon MVA python scripts

ikrav committed:
Further modifications to make possible to run different VID sequence for AOD and miniAOD for photons

Conflicts:
	PhysicsTools/SelectorUtils/python/tools/vid_id_tools.py

ikrav committed:
Deleted automatic production of photon MVA IDs that are too old. Left are the most recent 50ns ID (V2p1), 25ns ID (V2p1), and the new 80X ID

ikrav committed:
Fixed a problem with the order of MVA variables

ikrav committed:
Adjusted MD5 sum for the new MVA photon ID

ikrav committed:
Modified photon MVA for 80X: switched to tuning where for EE photon isolation is included in MVA after special scaling

ikrav committed:
Another adjustment of photon MVA configuration: new weights for the endcap, new working point cut values

ikrav committed:
Removed V1 of photon cut-based ID and introduced V2p2 cut-based photon ID which is a bettter tuning for 2016.

ikrav committed:
Changed the input variables rho and worst charged isolation to the form they are derived in the tuning of the latest photon MVA.

ikrav committed:
Updated pointer to indico slides for cut-based 80X photon ID

ikrav committed:
Updated MVA cut values for VID for 80X photon ID after modifying two MVA variables, as committed previously

ikrav committed:
Changes for complience with CMS coding stnadards: position of underscores, import statements, etc

ishvetso committed:
fix to run CITK isolation in VID during miniAOD production

Conflicts:
	PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

ikrav committed:
Replaced import * with explicit imports

Conflicts:
	RecoEgamma/EgammaIsolationAlgos/python/egmPhotonIsolationAOD_cff.py

ishvetso committed:
remove redefinition of pfNoPileUpCandidates

ishvetso committed:
switch to AOD format for switchOnVIDPhotonIdProducer()

ishvetso committed:
fix possible duplications with particleFlowTmpPtrs (added ONLY when the object is not allocated)

ikrav committed:
Removed duplicated import statement

ikrav committed:
A change for PF producer that calls EGM to match the changed naming in EGM python scripts

ikrav committed:
Changes to adjust the new photon ID code to use tasks rather than sequences to match recent changes in related packages in CMSSW

ikrav committed:
A few changes to imrove python coding without change in functioning

mtosi committed:
add HCAL DQM and Validation @HLT

mtosi committed:
fix DQMOffline/Trigger/python/DQMOffline_Trigger_cff.py

fix

fix

fix

fix

ikrav committed:
Fixed a bug of using abs that returns an int for the photon MVA

ikrav committed:
Added the latest 80X photon IDs to the default miniAOD production

ikrav committed:
Updated the photon cut-based ID version to the EGM-recommended one for 2016+ data

Commit:
updating FastjetJetProducer

mtosi committed:
fix

apana committed:
Make compatible with latest utm library (0.6.0 and later from Vienna)

Conflicts:
	L1Trigger/L1TGlobal/plugins/TriggerMenuParser.cc

Commit:
adding exceptions

Teemperor committed:
Don't forward declare CaloTower in CaloTowerFwd.h

This header won't compile with just a forward declaration of
CaloTower, because the Ref and RefVector types we declare will check the
ValueTrait of it's containing class, which is in this case CaloTower.
ValueTrait needs to have the definition of CaloTower to work, but
we only have a forward declaration in this header available.

This leads to undefined behavior, so we should replace this forward
declaration with an actual include.

People seem to anyway always include CaloTower too to
avoid this cryptic error, so I don't think this will hit compilation
times that much.

Commit:
adding exceptions

cmsbuild commented:
A new Pull Request was created by  @popovvp  for CMSSW_9_1_X.

It involves the following packages:

DQM/CTPPS


@cmsbuild, @dmitrijus, @vanbesien, @davidlange6 can you please review it and eventually sign? Thanks.
@davidlange6 you are the release manager for this.

cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

Commit:
adding customize process for FastjetJetProducer

Commit:
fixing custominzeHLTforCMSSW

Commit:
fixing RecoHi

Commit:
fixing puCenters

Sam-Harper committed:
initial version

rovere committed:
Pixel Tracking for 2017 online Beamspot

gartung committed:
fix gcc700 warning: class  has virtual functions and accessible non-virtual destructor [-Wnon-virtual-dtor]

rovere committed:
Add siStrip clusters

rovere committed:
Add siStrip clusters

kpedro88 committed:
use flavor 7 w/ type 15 for premixed MC packing/unpacking of QIE8 w/ uHTR (preserve error bits)

kpedro88 committed:
fix typo

kpedro88 committed:
remove unfinished packer attempt in HcalUHTRData

Commit:
updating branch

Commit:
removing parameters from VoronoiSubtractor

Commit:
fixing

Sam-Harper committed:
refactor for readability and fill descriptions

Sam-Harper committed:
updating to use VID and generic trigger event

Sam-Harper committed:
updating to use VID and generic trigger event

slehti committed:
HLTTauDQM: fixed typos in t&p mu+tau50 and ele+tau 2017 paths

cerminar committed:
add relval 1010 for EcalPedestals PCL

cerminar committed:
add workflow for pixel cluster couting AlCaRecos

Commit:
fix phase 1 HIMuon Initial step confusion between CA and non-CA doublet list (currently non-CA)

mtosi committed:
TopFolderName_ --> topFolderName_

fix

ikrav committed:
Several changes to follow recommendations of code reviewers

Commit:
Adding anode DB suppression status to the rechits

ikrav committed:
A fix to make the code compile related to the previous commit

leggat committed:
Adding in protection against attempts to access a non-existent ME and against not booking plots in the dqmEndLuminosityBlock method

albertdow committed:
adding the latest PFJets/Tau cross cleaning module

davidlange6 committed:
Merge pull request #18815 from forthommel/ctpps-runranges_patch2-9_2_X

CTPPS: Small memory footprint reduction for the channels mapping handler

Dr15Jones committed:
Avoid TClass::GetClass calls when data product missing

Calling TClass::GetClass can lead to lots of memory hoarding inside
ROOT. Therefore we should use our pre-cached TClass pointer from
BranchDescription.

yimuchen committed:
Update SiPixelPhase1RawData_cfi.py

cmsbuild committed:
Merge pull request #18998 from Dr15Jones/avoidTClassCalls

Avoid TClass::GetClass calls when data product missing

cmsbuild committed:
Merge pull request #18933 from gartung/Align-gcc700-dtor-warning-fix

Alignment : fix gcc700 warning: class  has virtual functions and accessible non-virtual destructor [-Wnon-virtual-dtor]

boudoul committed:
Use non-deprecated phase2 scenario

cmsbuild committed:
Merge pull request #18928 from gartung/Formats-gcc700-dtor-warning-fix

CalibFormats : fix gcc700 warning: class  has virtual functions and accessible non-virtual destructor [-Wnon-virtual-dtor]

davidlange6 committed:
Merge pull request #18934 from gartung/Calo-gcc700-dtor-warning-fix

CalibCalorimetry + CaloOnlineTools : fix gcc700 warning: class  has virtual functions and accessible non-virtual destructor [-Wnon-virtual-dtor]

thomreis committed:
Clean up memory from pointer.

slehti committed:
HLTTauDQM: fixed two mu+tau paths missing 'eta2p1'

davidlange6 committed:
Merge pull request #18638 from mtosi/numberPerLSFilter

add NumberOfEventsPerLS EDFilter

cmsbuild committed:
Merge pull request #19006 from boudoul/fixunitgeom

Use non-deprecated phase2 scenario in detector description unit tests

Commit:
Add PV information for DIP publication

albertdow committed:
use deltaR2 and consistent member data convention

cmsbuild committed:
Merge pull request #18939 from gartung/GeneratorInterface-gcc700-dtor-warning-fix

GeneratorInterface : fix gcc700 warning: class  has virtual functions and accessible non-virtual destructor [-Wnon-virtual-dtor]

cmsbuild committed:
Merge pull request #18935 from gartung/CondCore-gcc700-dtor-warning-fix

CondCore : fix gcc700 warning: class  has virtual functions and accessible non-virtual destructor [-Wnon-virtual-dtor]

cerminar committed:
add a test of the AlCaPCCZeroBiasFromRECO and AlCaPCCRandomFromRECO alcarecos

mtosi committed:
fix

Dr15Jones committed:
Avoid making three copies of a large PSet in TrackingMonitor

TrackingMonitor and two of its helpers (TrackAnalyzer and
TrackBuildingAnalyzer) were making copies of the module's PSet.
This PSet was quite large and making three copies per Stream is
not good. Now the module caches the PSet ID and gets the PSet
from the Registry when needed and passes the PSet to the helpers
when they need it.

The helpers also had virtual functions even though no other classes
inherited from them. These were de-virtualized.

TrackAnalyzer was not a unique name (a module with the same name
exists) so the class was moved to the dqm namespace.

dimattia committed:
second bunch of comments addressed.

Commit:
Online DQM fix for ML applications.

Implements 'keepBackup' option in the file saver.
If the option is set, the backup files (which are usually made every 15
lumisections) will not be deleted once a new backup is made.

This way it is possible to retrieve 'history' of the online DQM application.

Also adds 'dt4ml' client - an example how this new option can be used.

davidlange6 committed:
quiet harvesting step

albertdow committed:
working on a new solution for PFJets Tau overlap

kpedro88 committed:
add fastsim digi aliases for HCAL upgrades

Dr15Jones committed:
Do not read config during Event processing

The conf_ variable is a nullptr during Event processing so can not
be used. Instead, we cache the value we need in the constructor.

cmsbuild committed:
Merge pull request #18991 from Dr15Jones/fixMemoryLinkInDQMOfflineTrigger

Fix memory leaks in DQMOffline/Trigger

cmsbuild committed:
Merge pull request #18966 from forthommel/ctpps-diamond_dqm_corr-9_2_X

CTPPS: small corrections in the diamond DQM

cmsbuild committed:
Merge pull request #18941 from gartung/L1Trigger-gcc700-dtor-warning-fix

L1Trigger : fix gcc700 warning: class  has virtual functions and accessible non-virtual destructor [-Wnon-virtual-dtor]

cmsbuild committed:
Merge pull request #18938 from gartung/EventFilter-gcc700-dtor-warning-fix

EventFilter : fix gcc700 warning: class  has virtual functions and accessible non-virtual destructor [-Wnon-virtual-dtor]

cmsbuild committed:
Merge pull request #18646 from abaty/Phase1QuadTrackingFirstVersion_4master3

First Round of Phase 1 Tracking changes for Heavy Ions

albertdow committed:
changed logic within overlap removal and removed using namespace

albertdow committed:
adding missed std:: for unique_ptr

cmsbuild committed:
Merge pull request #18711 from akhukhun/fixHFTPemulatorAndLUTscale

adjust HF TP emulator and compression LUTs

mseidel42 committed:
Implement comments from @davidlange6

cmsbuild committed:
Merge pull request #18517 from kkotov/o2oRemoveOldParser

Moving the Prescale ES producer to the new parser and deleting old XML parser

davidlange6 committed:
Merge pull request #18808 from davidlange6/earlyDelFaster

speed up earlyDeleteSettings_cff.py by avoiding function calls

yimuchen committed:
Reverted changes to digis occupancy (implemented in another PR)

yimuchen committed:
Reverted changes to digis occupancy (implemented in another PR)

slehti committed:
HLTTauDQM: a parameter went missing in moving from 2016 to 2017 paths, added back

davidlange6 committed:
Merge pull request #18447 from enochnotsocool/DigiOccRotate_91X

Digi occupancy plot rotated + FED range adjusted (91X)

davidlange6 committed:
Merge pull request #18364 from mtosi/hcalDQMandVALIDATION@HLT

add HLT HCAL DQM/Validation

cmsbuild committed:
Merge pull request #18955 from jhgoh/RPCTimeFix

RPCRecHits nan protection

davidlange6 committed:
trim the size of these luts

fwyzard committed:
recover original behaviour: do not write .jsndata files for empty lumisections

cmsbuild committed:
Merge pull request #18803 from Teemperor/FixMissingIncludesInstantiate.h

Added missing includes to Instantiate.h.

rovere committed:
Improve region for pixelTracks

rovere committed:
Center in Z the globalRegion

leggat committed:
Removing string look-ups for dead and ineff ROC trend plots

rovere committed:
Improve logic for output files location

rovere committed:
Improve global tracking region definition

mariadalfonso committed:
add landau for M2 HEP17-shape

Commit:
Correct the output tags

BenjaminRS committed:
Fixing a bug to not try to take a L2 track when no L2 is available

rovere committed:
Improve logic for output files location

arunhep committed:
MC GTs updated with BPix fixes Fed Cabling,Sim geometry, HCALSiPM Fixes

Commit:
asjust the changes of depth for all PhaseI and PhaseII scenario

kpedro88 committed:
update HBHE raddam parameters

Sam-Harper committed:
allowing VID to gracefully fail if the input collection is invalid

Sam-Harper committed:
allowing VID to gracefully fail if the input collection is invalid

dimattia committed:
use of const string& into helper functions

Commit:
Update ieta=16 depths

Sam-Harper committed:
merging in python changes

Commit:
Update DQM configuration files for relevant HCAL AlCaReco jobs

cmsbuild committed:
Merge pull request #19045 from fwyzard/HLTriggerJSONMonitoring_92x

recover original behaviour: do not write .jsndata files for empty lumisections

cmsbuild committed:
Merge pull request #18988 from Dr15Jones/fixMemoryLeakEgammaTowerIsolation

Fix thread local memory leak in EgammaTowerIsolation

cmsbuild committed:
Merge pull request #18990 from Dr15Jones/fixMemoryLeakInConversionLikelihoodCalculator

Fix memory leak in ConversionLikelihoodCalculator

cmsbuild committed:
Merge pull request #19016 from Dr15Jones/avoidCopyingPSetsTrackingMonitor

Avoid making three copies of a large PSet in TrackingMonitor

cmsbuild committed:
Merge pull request #19021 from gartung/Geometry-gcc700-dtor-warning-fix

Geometry : fix gcc700 warning: class  has virtual functions and accessible non-virtual destructor [-Wnon-virtual-dtor]

davidlange6 committed:
Merge pull request #19022 from davidlange6/harv

quieter harvesting step

cmsbuild committed:
Merge pull request #18993 from leggat/phase1SummaryUpdate92X

Updates to pixel phase 1 summary class

davidlange6 committed:
Merge pull request #18989 from Dr15Jones/fixMemoryLeakInL1TMuonEndCapForestESProducer

Fixed memory leak in L1TMuonEndCapForestESProducer

cmsbuild committed:
Merge pull request #18959 from jalimena/NoBPTXTriggerValidation92X

update EXO NoBPTX trigger paths validation for 2017 pp collisions

mmusich committed:
use native C++ types, use enum instead of string comparisons and use vectors against maps

davidlange6 committed:
Merge pull request #18739 from arizzi/triggerLabels910pre3

Pack triggerobject standalone and photons/muon updated packing for miniaod

davidlange6 committed:
Merge pull request #19035 from dmitrijus/dt4ml

Online DQM fix for ML applications.

mseidel42 committed:
More abs->std::abs

cmsbuild committed:
Merge pull request #18962 from thomreis/uGMT-l1-dqm-muon-upgrades

L1 muon online DQM uprades

mtosi committed:
fix pixel DQM

rmanzoni committed:
vtx smearing parameter for early 2017 MC

cmsbuild committed:
Merge pull request #18531 from dimattia/MonitorForMRH_new

DQM quality monitor for SiStrip Gain (G2) from Multi Run Harvesting

yimuchen committed:
Tweaked plot ranges for more descriptive axis title and binning

fabozzi committed:
inject wf without approving

cmsbuild committed:
Merge pull request #19037 from BenjaminRS/L3FilterQuickFix92X

Fixing a bug to not try to take a L2 track when no L2 is available

wddgit committed:
Remove service named EnableFloatingPointExceptions

This was discussed and decided at the Core meeting
on 30 May 2017. We do not want to use resources maintaining
this service for several reasons. First of all, as far as we
know no one uses or has used this service and it has been
around for more than 10 years. Its unit test is broken. It
does not work properly in the new multithreaded environment
and would be difficult to fix, the precision control
parts of it are obsolete with modern processors, and it
does not work properly with some architectures.

cmsbuild committed:
Merge pull request #18985 from Dr15Jones/fixMemoryLeakMultipleScatteringX0Data

Fixed memory leak in MultipleScatteringX0Data

mariadalfonso committed:
add landau for M3 HEP17-shape

davidlange6 committed:
Merge pull request #19067 from rekovic/pr92x-l1t-parser-compatible-from-utm-0.6.0

pr92 L1T parser compatible utm-0.6.0 and later

davidlange6 committed:
Merge pull request #19015 from cerminar/92X_alca_relvals_v2

[92X] RelVal tests for new alca workflows

davidlange6 committed:
Merge pull request #19047 from kpedro88/FixPremixAgain4

Fix HCAL premixing with QIE8 and uHTR

cmsbuild committed:
Merge pull request #18996 from igv4321/hf-db-anode-status

Adding anode DB suppression status to the rechits

davidlange6 committed:
Merge pull request #19059 from mtosi/fixPixelDQM

fix pixel DQM

mmusich committed:
Remove files made useless from AfterAbortGap -> AAG transition; bring back SiStripGainsPCLHarvester

Sam-Harper committed:
reverting changes to VID

Sam-Harper committed:
removing egammaHLT DQM from HI

davidlange6 committed:
Merge pull request #18838 from aperloff/fixSpacingJetCorrectorParametersHelper

Fix spacing by replacing all tabs with spaces.

davidlange6 committed:
Merge pull request #18645 from ikrav/egm_id_80X_v3_photons_rebasedTo_CMSSW_9_2_X_2017-05-08-1100

EGM-certified photon IDs for 2016 data: cut based and MVA

fabozzi committed:
cleanup

cmsbuild committed:
Merge pull request #18856 from intrepid42/PowhegHooksBB4L_92X

Powheg-bb4l FSR veto

gpetruc committed:
No AssociationVector<bool>

gpetruc committed:
Fix scheduling problems from PATTriggerProducer thinking it may need the TriggerResults of the PAT in which it is running

albertdow committed:
minor changes (parameters,deltaR2,guard string) in both header and source

albertdow committed:
changed deltaR2 function

cmsbuild committed:
Merge pull request #19062 from fabozzi/from-CMSSW_9_2_X_noapprovewf

Injection of workflows without approving them

cmsbuild committed:
Merge pull request #19074 from gartung/CommonTools-gcc700-dtor-warning-fix

CommonTools/RecoUtils : fix gcc700 warning: class  has virtual functions and accessible non-virtual destructor [-Wnon-virtual-dtor]

Commit:
Update the MinBias trigger path

davidlange6 committed:
Merge pull request #19036 from bsunanda/Run2-hcx128

Run2-hcx128: adjust the depth assignments in HB for all PhaseI and PhaseII scenario

davidlange6 committed:
Merge pull request #19039 from kpedro88/RadDam-7

update HBHE raddam parameters

davidlange6 committed:
Merge pull request #18866 from igv4321/sipm-nonlinearity-fix

Switching to full pulse nonlinearity for SiPMs

davidlange6 committed:
Merge pull request #19050 from rovere/updateOnlineBSWithPVInformationMaster

Update online bs with pv information master

fabozzi committed:
updating 2017 gensim recycling to 9_2_1

fabozzi committed:
fixing statistics of JpsiMuMu_Pt-8 2017 relval

ianna committed:
Move DD algorith plugins in plugin directory

ianna committed:
moved headers

perrotta committed:
restore one line that got lost while rebasing

ianna committed:
Merge Base to Core

perrotta committed:
Factorize fillDescriptions

perrotta committed:
fix typo

cmsbuild committed:
Merge pull request #19092 from ianna/dd-algorithm-move

Move DD Algorithm Plugins in a Plugin Directory

perrotta committed:
remove also sumRecHits

perrotta committed:
Revert "remove also sumRecHits"

This reverts commit 57a4259a134ce05121cf920f07a48817a663d709.

perrotta committed:
Use the correct PR nr in the HLT customization

Commit:
Enabling depth studies in Ecal crystal

mariadalfonso committed:
fix array

Teemperor committed:
Renamed CaloTowerFwd.h to CaloTowerDefs.h

Because CaloTowerFwd.h is not really forward declaring anything, we
rename it to CaloTowerDefs.h to better represent it's purpose.

fioriNTU committed:
fix some ranges/binning of plots

davidlange6 committed:
Merge pull request #19080 from gpetruc/no_assovec_bool_to_valuemap_cmssw921

PhysicsTools fixes for Muon Tag & Probe in 92X

mmusich committed:
fix CalibTracker.Configuration.Common.PoolDBEEsource_cfi to remove disclaimer warning

perrotta committed:
Style fixes, from DavidL

Teemperor committed:
Add vector include to EventVariables.h

The v_sector member is using std::vector, so we need the definition
in this header.

ianna committed:
Merge DD ExprAlgo to Core

Teemperor committed:
Add missing include to ostream to Track/VertexCategories.h

At the end of these files we provide a printing function for
std::ostream, but we also need to include the ostream header before
that.

Teemperor committed:
Add missing includes to CategoryCriteria.h

We use Event, EventSetup and ParameterSet in this header, but we
didn't include the necessary headers. This patch adds the missing
includes.

mariadalfonso committed:
activate the new M3 landau shape only when the GT use the new shape

fioriNTU committed:
adjusting also residuals plots

DryRun committed:
Propagate EDAnalyzer to EDProducer to DQMEDHarver changes to HcalOnlineHarvesting

perrotta committed:
Merged refs/pull/19098/head from repository cms-sw

davidlange6 committed:
Merge pull request #19077 from MilanoBicocca-pix/master

Vertex smearing parameter for early 2017 MC

davidlange6 committed:
Merge pull request #19054 from bsunanda/Run2-hcx131

Run2-hcx131 Update DQM configuration files for relevant HCAL AlCaReco jobs

cmsbuild committed:
Merge pull request #18967 from kpedro88/Phase1-HF12

Make FastSim work with QIE10/11

davidlange6 committed:
Merge pull request #19031 from davidlange6/calibluts

trim the size of these luts

cmsbuild committed:
Merge pull request #18965 from stahlleiton/OnlineDQM_2DChi2QT_92X

OnlineDQM: Add Chi2 2D Quality Test

perrotta committed:
Do not use addDefault if you want autogenerated cfi's to be parsed for HLT

cmsbuild committed:
Merge pull request #18963 from mmusich/92X_PVValidationUpdates

[92X] PrimaryVertexValidation Updates

cmsbuild committed:
Merge pull request #19114 from DryRun/CMSSW_9_2_1_hcaldqm-fix-online-harvesting

Propagate EDAnalyzer to EDProducer to DQMEDHarvester changes to HcalOnli…

davidlange6 committed:
Merge pull request #19065 from wddgit/removeEnableFloatingPointExceptionsService

Remove service named EnableFloatingPointExceptions

davidlange6 committed:
Merge pull request #19033 from bsunanda/Run2-hcx127

Run2-hcx127 Correct the output cff sepecification for 2 minimum bias paths

albertdow committed:
moving Rsquared to constructor and removing unnecessary includes

slava77devel committed:
initialize default wrap to false

perrotta committed:
restore @module_label, but remove it from fillDescriptions

cmsbuild committed:
Merge pull request #19116 from mmusich/92X_fixCalibTrackerCommonWarning

Remove warnings from CalibTracker ESSource provider

cmsbuild committed:
Merge pull request #19107 from Teemperor/FixMissingIncludeEventVariables.h

Add vector include to EventVariables.h

cmsbuild committed:
Merge pull request #19099 from bsunanda/Run2-sim08

Run2-sim08 Enabling depth studies in Ecal crystal

cmsbuild committed:
Merge pull request #19061 from enochnotsocool/FEDTrendFix

FED Digis trend plot fix

cmsbuild committed:
Merge pull request #19103 from fioriNTU/FixRangesPhase192X

fix some ranges/binning of plots in Phase 1 DQM

davidlange6 committed:
Merged refs/pull/19038/head from repository cms-sw

davidlange6 committed:
Merged refs/pull/19091/head from repository cms-sw

cmsbuild committed:
Merge pull request #19120 from slava77/CMSSW_9_2_1/sign921/vgCleanup-mix

initialize default wrap to false in MixingModule Adjuster

davidlange6 committed:
Merge pull request #19106 from ianna/dd-expralgo-cleanup

DD ExprAlgo Merge with Core

mtosi committed:
update FastTimerServiceClient

davidlange6 committed:
Merge pull request #18805 from Teemperor/FixCaloTowerFwd.h

Don't forward declare CaloTower in CaloTowerFwd.h

mtosi committed:
update directory for EndPath

ianna committed:
Migrate to multithreaded analyzers

mariadalfonso committed:
document the 205 shape

mtosi committed:
add tracking monitoring to online beam spot

albertdow committed:
removing initialisation of mindR_ and using only matchingR2_ instead

cmsbuild committed:
Merge pull request #19111 from Teemperor/FixIncludesCategoryCriteria.h

Add missing includes to CategoryCriteria.h

davidlange6 committed:
Merge pull request #19046 from Sam-Harper/GenericTPDQM920

E/gamma T&P Offline DQM : 92X + VID + GenericTrigEventFlag Changes

slehti committed:
HLTTauDQM: changed tagAndProbePaths to be a local parameter

cmsbuild committed:
Merge pull request #18997 from albertdow/FinalCrossCleaningModule

adding a PFJets-Tau overlap removal module for HLT

cmsbuild committed:
Merge pull request #19130 from ianna/geom-mt-analyzers

Use Thread-safe Analyzers

cmsbuild committed:
Merge pull request #19128 from mtosi/fixFastTimerServiceClient

update FastTimerServiceClient

cmsbuild committed:
Merge pull request #19132 from mtosi/trackingDQM4onlineBS_92x

add tracking monitoring to online beam spot

davidlange6 committed:
Merge pull request #19070 from mariadalfonso/siPMdataM2Shape_92X

HEP17: pulse shape update (M2,M3,M0)

cmsbuild committed:
Merge pull request #19075 from mmusich/92X_remove_unused_configuration

Remove files made useless from AfterAbortGap -> AAG transition

rmanzoni committed:
update Z because of cogging adjustment by LHC

davidlange6 committed:
Merge pull request #19127 from davidlange6/newPixGeom

New pixel geometry in runTheMatrix tests

davidlange6 committed:
Merge pull request #18172 from slehti/HLTTauValidationUpdate_CMSSW91X_April2017

Hlt tau validation update cmssw91 x april2017

davidlange6 committed:
Merge pull request #19109 from Teemperor/VertexCategories.h

Add missing include to ostream to Track/VertexCategories.h

davidlange6 committed:
Merge pull request #19137 from MilanoBicocca-pix/master

update Z because of cogging adjustment by LHC

cmsbuild committed:
Merge pull request #19098 from perrotta/CMSSW_9_2_X_FastjetJetProducerUpdateWithFactorizedFilldescriptions

Updating FastjetJetProducer and related producers with (factorized) fillDescriptions

popovvp committed:
Merge branch 'master' into ctpps_pixel_dqm4c

cmsbuild commented:
Pull request #18836 was updated. @perrotta, @ghellwig, @arunhep, @cerminar, @monttj, @cmsbuild, @rekovic, @franzoni, @slava77, @ggovi, @mulhearn, @davidlange6 can you please check and sign again.

perrotta commented:
-1
You merged the master branch in this 91X PR...
